src/app/containers/Header/index.stories.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import * as svgs from '@bbc/psammead-assets/svgs';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import { withKnobs } from '@storybook/addon-knobs';
+import HeaderContainer from '.';
+import { RequestContextProvider } from '../../contexts/RequestContext';
+import services from '../../lib/config/services';
+import { ServiceContextProvider } from '../../contexts/ServiceContext';
+
+storiesOf('Header Container', module)
+  .addDecorator(withKnobs)
+  .add(
+    'default',
+    inputProvider(
+      null,
+      ({ service }) => {
+        const serviceContextStub = {
+          product: 'BBC News',
+          service,
+          brandSVG: svgs[service],
+        };
+        return (
+          <ServiceContextProvider service={service} value={serviceContextStub}>
```

Comment:
ServiceContextProvider doesnt take value, do you mean ServiceContext.Provider? if so, it doesnt take service

Reply:
i see,  updating now to use ServiceContext.Provider as the container does not require service prop anyways